packages/api/endpoints/executions.js

```diff
@@ -261,8 +283,95 @@ async function workflowsByGranules(req, res) {
   return res.send(workflowNames);
 }
 
+/**
+ * Deletes execution records in bulk for a specific collection.
+ *
+ * Does *not* publish execution deletion event to SNS topic
+ *
+ * @param {Object} req - The request object.
+ * @param {Object} req.params - The request parameters.
+ * @param {Object} req.body - The request body.
+ * @param {number|string} [req.body.batchSize=5000] - The number of records to delete in each batch.
```

Comment:
This annotation needs fixed - this was split into es/db batch.

Reply:
TODO